docs/UserGuide.md

```diff
@@ -65,9 +65,9 @@ This guide serves to provide a reference for first-time users to get familiar wi
 
 ### Viewing help: `help`
 
-Shows a message explaining how you can access the help page.
+Shows a message as shown in *Figure 2* explaining how you can access the help page.
```

Comment:
Small nitpick: perhaps you can add the word "below" after figure 2 to standardise with the figure 1 description above.